apps/mobile/src/libs/enums/fonts/fonts-families.enum.ts

```diff
@@ -0,0 +1,11 @@
+const FontFamilies = {
```

Comment:
Let's change it to
```
  const fontWeightToFamilyMap = {
  		bold: "Nunito-Bold",
		  extraBold: "Nunito-ExtraBold",
		  medium: "Nunito-Medium",
		  regular: "Nunito-Regular",
		  semiBold: "Nunito-SemiBold",
  };
```
and move directly to `Text` component.

Reply:
Done